tools/boards.txt.py

```diff
@@ -1153,7 +1153,10 @@ def flash_map (flashsize_kb, spiffs_kb = 0):
     else:
         max_upload_size = 1024 * 1024 - reserved
         spiffs_start = (flashsize_kb - spiffs_kb) * 1024
-        spiffs_blocksize = 8192
+        if spiffs_kb < 512:
```

Comment:
The problem introduced here is you have made existing small SPIFFS filesystems unreadable (and probably will cause hangs/crashes when they're attempted to be mounted).

Reply:
I went thru the logic and on review it seems OK.

This only applies to existing @M or greater SPIFFS chips whose FS was <512KB

4M and above do not support 512K SPIFFS.

You only just added <512KB to these 2-3 commits ago, so it's never been released.

Therefore, this only changes 2M(<512K) spiffs, and it's your addition less than a week ago.  I say it's OK by me!